addons/mass_mailing/models/mailing.py

```diff
@@ -988,7 +991,7 @@ def _get_recipients(self):
         res_ids = self.env[self.mailing_model_real].search(mailing_domain).ids
 
         # randomly choose a fragment
-        if self.ab_testing_enabled and self.ab_testing_pc < 100:
+        if self.ab_testing_enabled and not self.ab_testing_is_winner_mailing:
```

Comment:
I don't quite understand this change? This did not seem incorrect even for your "special use case".

Reply:
Mm, yeah I guess the two are equivalent here if we're checking ``ab_testing_enabled`` anyway. I guess the point is that it's more explicit to use the field, doesn't seem to make much sense without the context of knowing "100% ab_testing = winner"